app/logic/bitstore.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
```

Comment:
my suggestion is that the bitstore should actually not move into logic but should get its own little module. It is like a db model really -- or even a wrapper around the DB. The only parts of bitstore that should be in logic are those actually involving business logic (i'm not sure there are only).

summary: i suggest bitstore go to app/bitstore.py

NOTE: happy to merge this request and make this change as one additional refactor.